include/dxc/dxcapi.h

```diff
@@ -298,6 +298,13 @@ IDxcVersionInfo : public IUnknown {
   virtual HRESULT STDMETHODCALLTYPE GetFlags(_Out_ UINT32 *pFlags) = 0;
 };
 
+#ifdef SUPPORT_QUERY_GIT_COMMIT_INFO
```

Comment:
It's probably OK to not guard the interface declaration - whether the thing is implemented or not is a runtime thing that the client will discover via QI.

Reply:
Yep. Done.